webapp/channels/src/components/post_edit_history/edited_post_item/edited_post_item.test.tsx

```diff
@@ -46,10 +49,16 @@ describe('components/post_edit_history/edited_post_item', () => {
         expect(wrapper).toMatchSnapshot();
     });
     test('clicking on the restore button should call openRestorePostModal', () => {
-        const wrapper = shallow(<EditedPostItem {...baseProps}/>);
+        const wrapper = shallow(
+            <CompassThemeProvider theme={{}}>
```

Comment:
You don't need this provider because the component itself actually has one. The problem with this test though is that `baseProps.theme` is currently an empty object which is invalid.

If you change `baseProps.theme` to one of the real themes like `Preferences.THEMES.denim`, this test looks like it'll work.

Reply:
Alternatively, if you wanted to convert these tests to use React Testing Library since I see that you tried that at some point, we'd appreciate that. It's certainly not necessary though, but I thought I'd mention it since we'll be doing that with all the tests eventually. If you do though, the `renderWithContext` helper from `tests/react_testing_utils` would probably be very helpful for that.

Feel totally free to ignore this though